compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolResolver.java

```diff
@@ -258,6 +262,14 @@ private boolean hasSameOwner(BSymbol symbol, BSymbol foundSym) {
                 ((foundSym.owner.tag & SymTag.INVOKABLE) == SymTag.INVOKABLE);
     }
 
+    private boolean isDependentTestPackage(BSymbol symbol, BSymbol foundSym) {
```

Comment:
Shall we rename this? It is not  `isDependentTestPackage`, rather it validates, whether found symbol defined in the parent package. Shall we say 'isRedeclaredInParentModule' ?